compose/src/main/java/com/example/compose/MainActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.compose
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import com.example.compose.navigation.SoptNavGraph
+import com.example.compose.ui.theme.INSOPTAndroidPracticeTheme
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MainActivity : ComponentActivity() {
```

Comment:
```suggestion
class MainActivity : AppCompatActivity() {
```

이렇게 하셔도 되긴합니다~(취사선택)

Reply:
쿠쿠~

Reply:
ㅋㅋ 쿠쿠 킹받네요